templates/parachain/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+// The parachain-template crate helps with keeping the README.md in sync
+// with code sections across the components under the template: node,
+// pallets & runtime, by using `docify`.
```

Comment:
nit, maybe:
```suggestion
+ //! The parachain-template crate exists solely to keep the README.md file up-to-date using docify.
//! It ensures synchronization between the README and code sections across the template components,
//! such as the node, pallets, and runtime. Note that this crate itself is not copied into the
//! template repository.
```

Reply:
@seemantaggarwal I believe it is a valuable improvement, clear doc why this crate was created is always appreciated. Please re-consider including this.